content/yaml-quick-start/building-a-flutter-app.md

```diff
@@ -197,9 +197,9 @@ workflows:
         - other # <-- (Includes APP_STORE_ID)
         # Add the above group environment variables in Codemagic UI (either in Application/Team variables) - https://docs.codemagic.io/variables/environment-variable-groups/
       vars:
-        XCODE_WORKSPACE: "Runner.xcworkspace"
-        XCODE_SCHEME: "Runner"                
-        BUNDLE_ID: "YOUR_BUNDLE_ID_HERE"
+        XCODE_WORKSPACE: 'Runner.xcworkspace'
+        XCODE_SCHEME: 'Runner'
+        BUNDLE_ID: 'YOUR_BUNDLE_ID_HERE'
```

Comment:
just wondering about these changes, could there be a difference between using double and single quotation marks?